### PR TITLE
25 restapi

### PR DESCRIPTION
**Pull request by joergrs**
Solves #25

### CODE REVIEW

File: app/data/ops.py [hunk 1]
```diff
@@ -35,6 +36,8 @@ def __init__(self):
 
         self.Session = sessionmaker()
         self.Session.configure(bind=engine)
+        if self.MAIN_SESSION is None:
+            self.MAIN_SESSION = self.Session()
 
     def add_new_image(self, file, label_names):
```
Comment (liephat) [inline app/data/ops.py:42]: Turn into static methods?

File: app/data/ops.py [hunk 2]
```diff
@@ -92,3 +95,6 @@ def filelist(self):
         """
         with auto_session(self.Session) as session:
             return [file for (file,) in session.query(Image.file).all()]
+
+    def all_images(self):
```
Comment (liephat) [inline app/data/ops.py:99]: this also

File: app/data/ops.py [hunk 3]
```diff
@@ -97,4 +97,7 @@ def filelist(self):
             return [file for (file,) in session.query(Image.file).all()]
 
     def all_images(self):
-            return self.MAIN_SESSION.query(Image).all()
+        return self.MAIN_SESSION.query(Image).all()
+
+    def all_labels(self):
```
Comment (liephat) [inline app/data/ops.py:102]: and this

**Review by liephat — APPROVED** [review]

File: app/data/ops.py (hunk 2 above)
Comment (joergrs) [reply app/data/ops.py]: check

File: app/data/ops.py (hunk 3 above)
Comment (joergrs) [reply app/data/ops.py]: check

**Comment by joergrs** [comment]
closes #25

**Pull request merged**